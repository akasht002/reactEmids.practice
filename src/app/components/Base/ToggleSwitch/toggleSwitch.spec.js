import React from 'react';
import {ToggleSwitch} from './index';
import renderer from 'react-test-renderer';

 test('ToggleSwitch Component Testing', () => {
  const component = renderer.create(
    <ToggleSwitch></ToggleSwitch>,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});