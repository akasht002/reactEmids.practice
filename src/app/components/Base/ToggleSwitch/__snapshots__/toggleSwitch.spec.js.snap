// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ToggleSwitch Component Testing 1`] = `
<label
  className="switch"
>
  <input
    type="checkbox"
  />
  <span
    className="slider round"
  />
</label>
`;
